package/cfg/rke2-cis-1.23-hardened/policies.yaml

```diff
@@ -93,7 +93,8 @@ groups:
 
       - id: 5.2.2
         text: "Minimize the admission of privileged containers (Manual)"
-        audit: "kubectl get psp global-restricted-psp -o json | jq -r '.spec.runAsUser.rule'"
+        #audit: "kubectl get psp global-restricted-psp -o json | jq -r '.spec.runAsUser.rule'"
+        audit: kubectl get psp global-restricted-psp && kubectl get psp global-restricted-psp -o json | jq -r ".spec.runAsUser.rule" || kubectl get psp restricted-noroot-psp -o json | jq -r ".spec.runAsUser.rule"
```

Comment:
I might have missed something, but I don't think that this test and the other one will properly work. For example, I created a RKE2 v1.24 cluster with the `restricted-noroot-psp` and not RKE2 default PSP `global-restricted-psp`. So the cluster will create `restricted-noroot-psp`, but not `global-restricted-psp`. If I execute the command locally, it will fail:

```shell
> kubectl get psp global-restricted-psp -o json | jq -r .spec.requiredDropCapabilities[] || kubectl get psp restricted-noroot-psp -o json | jq -r .spec.requiredDropCapabilities[] 
Warning: policy/v1beta1 PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
Error from server (NotFound): podsecuritypolicies.policy "global-restricted-psp" not found
```